schedule.py

```diff
@@ -27,7 +27,7 @@ def add_job(cron, backup_script, test_job):
 def schedule_backups(test_job):
     here = dirname(abspath(__file__))
     print("Scheduling backup task")
-    backup_script = join(here, "backup.py")
+    backup_script = "cd {} && {}".format(here, join(here, "backup.sh"))
```

Comment:
I would prefer it if the cd logic was in `backup.sh` because otherwise every time we call it we need to make sure we're in the right directory